metrictank.go

```diff
@@ -102,6 +102,28 @@ func init() {
 func main() {
 	startupTime = time.Now()
 
+	/***********************************
+		Initialize Logger
+	***********************************/
+	log.NewLogger(0, "console", fmt.Sprintf(`{"level": %d, "formatting":false}`, logLevel))
```

Comment:
It seems like this will bypass the `log-level` config setting for this console logger. Might it be better to use `fmt` to output the error trace until the config is loaded?

Reply:
Bypasses log-level?

golang's init function runs first so log-level will be parsed already

Reply:
log-level is also in the config file and overrideable via envvars. That doesn't happen until `flag.Parse()` below.